utils/helpers/checks.ts

```diff
@@ -9,6 +9,10 @@ export const isEntrepreneurIndividuelFromNatureJuridique = (
   natureJuridique: string
 ) => ["1", "10", "1000"].indexOf(natureJuridique) > -1;
 
+export const isNotPersonneMoraleFromNatureJuridique = (
+  natureJuridique: string
+) => natureJuridique.startsWith("2");
```

Comment:
You could even use 
`isSocietePersonnePhysiqueFromNatureJuridique`
And as `isPersonneMorale` is almost always used with a negation, prefer `isPersonnePhysique` as it makes more sense in the context